imagefiles/entrypoint.sh

```diff
@@ -35,6 +35,11 @@ if [[ -n $BUILDER_UID ]] && [[ -n $BUILDER_GID ]]; then
         /dockcross/pre_exec.sh
     fi
 
+    # Execute project specific pre execution hook
+    if [[ -e /work/.dockcross ]]; then
+       /work/.dockcross
```

Comment:
Since it is executed in the current directory belonging to the user ... I will go with this approach instead.